src/app/archive/create_schema.sql

```diff
@@ -39,15 +39,15 @@ CREATE UNIQUE INDEX receipt_chain_hash_id ON receipt_chain_hashes (hash);
 
 CREATE TABLE user_commands (
   id serial PRIMARY KEY,
```

Comment:
is "serial" the id type in postgres?

Reply:
yes. Serial automatically increments an an id for you when you create a new row.